packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -206,6 +206,9 @@ export const addAuction = async (
   );
 
   const governedContractKits = await governedContractKitsP;
+  // @ts-expect-error The original auctioneerKit had everything it needs
+  await governedContractKits.init(legacyKit.instance, legacyKit);
```

Comment:
I don't think this should have an `await`.. init() is sync.

I imagine this clause could move earlier, so we're stashing the old value before we reset/resolve the new value. That would make more temporal sense, but 1: since our promise-space fulfills `legacyKit` to the old value even though/if we've reset/resolved the matching `produceAuctioneerKit`, it doesn't matter which order we do it in, and 2: doing it "backwards" makes it more obvious that we know the promise-space works that way; it doesn't look like we're accidentally relying on it *not* giving us back the new value. maybe.